JASP-Common/utils.cpp

```diff
@@ -399,3 +401,36 @@ std::string Utils::doubleToString(double dbl)
 	conv << dbl;
 	return conv.str();
 }
+
+// hex should be 4 hexadecimals characters
+std::string Utils::_convertEscapedUnicodeToUTF8(const std::string& hex)
```

Comment:
If you change the argument to `std::string hex` you do not need to define `std::string results = hex;` anymore.